imports/include/niFakeNonIvi.h

```diff
@@ -1,9 +1,10 @@
 #ifndef NIFAKE_NON_IVI_H
 #define NIFAKE_NON_IVI_H
 #include <cstdint>
+// For CVIAbsoluteTime
```

Comment:
I'm not thrilled about this since we were trying to avoid it, but we need the definition of CVIAbsoluteTime, and copy/pasting it in this header file meant I also had to pull in some other things like uInt64 and such. If you feel strongly about it I can go down that route again and see how bad it looks...

Reply:
I think either way is ok. I guess you could also have a variant of CVITime that uses uint64_t/etc.